src/chroot_script

```diff
@@ -212,6 +212,25 @@ echo "mapping wlan1" >> /etc/network/interfaces
 echo "  script /root/bin/map_iface" >> /etc/network/interfaces
 echo "source /boot/octopi-network.txt" >> /etc/network/interfaces
 
+# allow wifi network configuration via /boot/octopi-wifi.txt
+mv /etc/wpa_supplicant/wpa_supplicant.conf /boot/octopi-wifi.txt
+ln -s /boot/octopi-wifi.txt /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
Tested it out. Raspbian Jessie does move `/boot/wpa_supplicant.conf` -> `/etc/wpa_supplicant/wpa_supplicant.conf`. However, it does it only on the **1st** boot. 

It doesn't solve our problem. I, as well as my friends, move our 3D printers to different places. Meetups, trade shows, etc. We want the ability to adding new wifi network when we move our 3D printers to a new place, so that we can use OctoPrint wherever we go.

It looks like the most straightforward way is to use symlink `/boot/octopi-wifi.txt` to `/etc/wpa_supplicant/wpa_supplicant.conf`. I can't use the name `/boot/wpa_supplicant.conf` as it'll trigger Raspbian to move it to folder `/etc/wpa_supplicant` on the 1st boot.

I revised the PR accordingly, including the comments you suggested. Let me know if it's ok to you.